build.sh

```diff
@@ -27,12 +27,13 @@ if test ! -d packages/KoreBuild; then
     mono .nuget/nuget.exe install Sake -version 0.2 -o packages -ExcludeVersion
 fi
 
-if ! type k > /dev/null 2>&1; then
-    source packages/KoreBuild/build/kvm.sh
+if ! type dnvm > /dev/null 2>&1; then
```

Comment:
While you're updating all the repos, also update the build files in Universe\build-template. @troydai's updated it recently, but the contents of this file look more correct